mkt/site/monitors.py

```diff
@@ -240,7 +241,7 @@ def package_signer():
         msg = 'Error on package signing (%s): %s' % (destination, e)
         return msg, msg
     finally:
-        storage.delete(signed_path)
+        local_storage.delete(signed_path)
```

Comment:
I wonder if it's worth passing in a kwarg to tell it to use `local_storage` instead of these? It seems inefficient to copy it to S3 just to delete it right after.